script/functions.sh

```diff
@@ -103,6 +98,16 @@ function run_spec_suite_for {
   fi;
 }
 
+function run_spec_suite_for_rspec_rails {
+  echo "Running specs for rspec-rails"
+  pushd ../rspec-rails
+  unset BUNDLE_GEMFILE
+  unset RUBYOPT
+  travis_retry eval "(unset RUBYOPT; exec bundle install)"
+  bundle exec rspec spec --backtrace --format progress --profile --format progress --out $SPECS_HAVE_RUN_FILE
```

Comment:
@JonRowe I've tried many options, and this one, the most straightforward, is the only one that worked.

Without `unset RUBYOPT` Rails is complaining that [`uninitialized constant Gem`](https://github.com/rspec/rspec-core/runs/1513144181?check_suite_focus=true).
With `--binstubs --standalone --path=../bundle` and subsequent `bin/rspec` the [binstub apparently ceases to find ../bundle](https://github.com/rspec/rspec-core/runs/1513825875?check_suite_focus=true):

```
# Gem::MissingSpecError:
#   Could not find 'sqlite3' (~> 1.4) among 30 total gem(s)
#   Checked in 'GEM_PATH=/home/runner/.gem/ruby/2.5.0:/opt/hostedtoolcache/Ruby/2.5.8/x64/lib/ruby/gems/2.5.0', execute `gem env` for more information
```

Would you agree for such a change?
 - Yes, surely we're shaving some fractions of a second by using `standalone`, but is it worth such a complication?
 - `SPECS_HAVE_RUN_FILE` is quite an odd flag
 - `[ -d ../$1 ]` is useless - we do clone it, and if we don't - we don't `run_spec_suite_for` for this dir

To my huge surprise, the problem with `uninitialized constant Gem` can be reproduced with [a small single commit](https://github.com/rspec/rspec-core/commit/ddbb2d0a52c0ee88f8940378fbcc2d6f1a2ebe02) comparing to the green 4-0-dev. That very commit that drops `.travis.yml` and barely does anything else.

Reply:
I think its the `ffi` version, I had to fix rspec-core for main too, I've applied that patch to `4-0-dev` and opened simplified PRs to see how they go

Reply:
Awesome, thanks!
I'll close this series for now, we can address the removal of .travis.yml later.